pkg/standalone/list.go

```diff
@@ -156,3 +160,16 @@ func getIntArg(argMap map[string]string, argKey string, argDef int) int {
 	}
 	return argDef
 }
+
+// GetCLIPIDCountMap returns a map of CLI PIDs to number of apps started with it.
+func GetCLIPIDCountMap() map[int]int {
+	cliPIDCountMap := make(map[int]int)
+	apps, err := List()
+	if err != nil {
+		return nil
```

Comment:
why is the error swallowed here?

Reply:
this error was not that useful here because we do not want the stop to exit here and again the same logic to get the list of apps was there in `Stop` method, which would have handled the failure case.

So, refactored `Stop` method a little to avoid the duplication of logic. Let me know your thoughts on this.

Reply:
Ok ... but have the List() queried only once then and use if for either normal stop or stop run template ...